package.json

```diff
@@ -31,6 +31,7 @@
     "webpack-cli": "^3.1.2"
   },
   "dependencies": {
+    "fake-indexeddb": "^2.0.4",
```

Comment:
Please move this to the `devDependencies` section, as it is not required during the lib execution.

Reply:
done